edgelet/iotedge/src/init/import/mod.rs

```diff
@@ -253,7 +253,7 @@ fn execute_inner(
     ) = {
         let old_config::Provisioning {
             provisioning,
-            dynamic_reprovisioning,
+            dynamic_reprovisioning: _,
```

Comment:
Add a comment on this line along the lines of `// TODO: Migrate this to edged config when support for dynamic reprovisioning is reinstated in edged.`

@massand Do you have a work item for it already?

Reply:
added the comment